contrib/imgui/imgui.h

```diff
@@ -2305,6 +2305,8 @@ struct ImGuiIO
     ImWchar16   InputQueueSurrogate;                // For AddInputCharacterUTF16()
     ImVector<ImWchar> InputQueueCharacters;         // Queue of _characters_ input (obtained by platform backend). Fill using AddInputCharacter() helper.
 
+    bool        ImGuiIgnoreClampWindow;
```

Comment:
this _is_ ImGui code, so `IgnoreClampWindow` would be sufficient

Reply:
fixed.